packages/cms-cli/commands/fetch.js

```diff
@@ -27,56 +27,91 @@ const {
 } = require('../lib/usageTracking');
 
 const COMMAND_NAME = 'fetch';
+const DESCRIPTION =
+  'Fetch a file, directory or module from HubSpot and write to a path on your computer';
 
-function configureFetchCommand(program) {
-  program
-    .version(version)
-    .description(
-      'Fetch a file, directory or module from HubSpot and write to a path on your computer'
+const action = async ({ src, dest }, options = {}) => {
+  setLogLevel(options);
+  logDebugInfo(options);
+
+  const { config: configPath } = options;
+  loadConfig(configPath);
+  checkAndWarnGitInclusion();
+
+  if (
+    !(
+      validateConfig() &&
+      (await validatePortal(options)) &&
+      validateMode(options)
     )
+  ) {
+    process.exit(1);
+  }
+
+  if (typeof src !== 'string') {
+    logger.error('A source to fetch is required');
+    process.exit(1);
+  }
+
+  dest = resolveLocalPath(dest);
+
+  const portalId = getPortalId(options);
+  const mode = getMode(options);
+
+  trackCommandUsage(COMMAND_NAME, { mode }, portalId);
+
+  // Fetch and write file/folder.
+  downloadFileOrFolder({ portalId, src, dest, mode, options });
+};
+
+// Yargs Configuration
+const command = `${COMMAND_NAME} <src> [dest]`;
+const describe = DESCRIPTION;
+const builder = yargs => {
+  addConfigOptions(yargs, true);
+  addPortalOptions(yargs, true);
+  addLoggerOptions(yargs, true);
+  addOverwriteOptions(yargs, true);
+  addModeOptions(yargs, { read: true }, true);
+
+  yargs.positional('src', {
+    describe: 'Path in HubSpot Design Tools',
+    type: 'string',
+    demand: true,
+  });
+
+  yargs.positional('dest', {
+    describe:
+      'Path to the local directory you would like the files to be placed, relative to your current working directory. If omitted, this argument will default to your current working directory',
```

Comment:
I wonder if we should shorten this a bit. For me, the readability starts to suffer when it wraps across three lines.

![image](https://user-images.githubusercontent.com/266036/90671125-41449300-e222-11ea-9ff9-996de1c29379.png)

Reply:
Sure I can do that.  I was just trying to keep consistency between the CLI and docs

Reply:
:+1: FWIW, I don't think addressing this is a blocker.

Longer term, we probably should think about some sort of way to generate the reference documentation. Perhaps there also is a way to adjust how `yargs` formats help so that multi-line descriptions work better.

Reply:
Shortened it up a bit